File: Sem1/Assignment/code/LBEP_Ex4.php
<?php
	echo "***************************************\n";
	echo "\tPHP PROGRAM LANGUGE\n";
	echo "\t1.Question 02.\n";
	echo "\t2.Question 03.\n";
	echo "\t3.Exit\n";
	echo "***************************************\n";
	$choose = readline("Enter Choice (1 - 3): ");
	if($choose == 1){
		$x = readline("Enter the width of Rectangle: ");
		$y = readline("Enter the high of Rectangle: ");
		$per = ($x + $y)*2;
		$area = $x * $y;
		echo "\tPerimeter of the Rectangle: 2 * ({$x} + {$y}) = {$per}\n";
		echo "\tArea of of the Rectangle: {$x} * {$y} = {$area}\n";
	} elseif($choose == 2){
		$m = readline("Enter the radius of Circle: ");
		$per_c = $m *2 *3.14;
		$area_c = $m*$m*3.14;
		echo "\tPerimeter of the Circle: 2 * {$m} *3.14 = {$per_c}\n";
		echo "\tArea of the Circle: {$m} * {$m} *3.14 = {$area_c}\n";
	}else{
		echo "\tGoodbye\n";
	}

File: Sem1/Assignment/code/LBEP_Ex4.java
import java.util.*;

public class LBEP_Ex4{
	public static void main(String[] args){
		Scanner input =  new Scanner(System.in);
		System.out.println("*****************************");
		System.out.println("\tJava Program language");
		System.out.println("\t1. Question 02.");
		System.out.println("\t2. Question 03.");
		System.out.println("\t3. Exit.");
		System.out.println("******************************");
		System.out.println("Enter choice: (1-3):");
		String str = input.next();
		int num = Integer.parseInt(str);
		if(num == 1){
			System.out.print("Enter the width of Rectangle: ");
			double x = Double.parseDouble(input.next());
			System.out.print("Enter the high of Rectangle: ");
			double y = Double.parseDouble(input.next());
			System.out.printf("\tPerimeter of the Rectangle: 2 * (%.1f + %.1f) = %.1f\n", x,y,2*(x+y));
			System.out.printf("\tArea of the Rectangle: %.1f * %.1f = %.1f\n",x,y,x*y);
		} else if(num == 2){
			System.out.print("Enter the radius of Circle: ");
			double z = Double.parseDouble(input.next());
			System.out.printf("\tPerimeter of the Circle: 2 * %.2f *3.14 = %.2f\n", z, 2*z*3.14);
			System.out.printf("\tArea of the Circle: %.2f * %.2f *3.14 = %.2f\n",z,z*z*3.14);
		}else{
			System.out.println("Goodbye");
		}
	}

}
